geode-core/src/main/java/org/apache/geode/management/internal/beans/MemberMBeanBridge.java

```diff
@@ -193,12 +192,8 @@
 
     dm = system.getDistributionManager();
```

Comment:
TODO: get rid of `dm` field and just use the `system` field directly instead of fetching from dm within `shutDownMember()`

Reply:
Done